apps/sports-helsinki/src/domain/search/combinedSearch/adapters/VenueSearchAdapter.ts

```diff
@@ -80,6 +82,11 @@ class VenueSearchAdapter implements CombinedSearchAdapter<VenueSearchParams> {
           ? SortOrder.Descending
           : SortOrder.Ascending,
       } as OrderByDistance;
+    } else if (
+      input.venueOrderBy &&
+      isAccessibilityProfile(input.venueOrderBy)
+    ) {
+      this.orderByAccessibilityProfile = input.venueOrderBy;
     }
```

Comment:
If the suggested changes in https://github.com/City-of-Helsinki/events-helsinki-monorepo/pull/519/files#r1365036674 are taken into account then this can be simplified:
```suggestion
    } else if (isAccessibilityProfile(input.venueOrderBy)) {
      this.orderByAccessibilityProfile = input.venueOrderBy;
    }
```